packages/perseus/src/util/parse-perseus-json/perseus-parsers/perseus-image-background.ts

```diff
@@ -3,21 +3,26 @@ import {
     number,
     object,
     optional,
+    pipeParsers,
     string,
     union,
 } from "../general-purpose-parsers";
+import {stringToNumber} from "../general-purpose-parsers/string-to-number";
 
 import type {Parser} from "../parser-types";
 import type {PerseusImageBackground} from "@khanacademy/perseus";
 
+const numericToNumber = pipeParsers(union(number).or(string).parser).then(
+    stringToNumber,
+).parser;
```

Comment:
I don't understand why we need to `union` `number` or `string` and then pass that result through `stringToNumber`.  Doesn't `stringToNumber` handle parsing a string or number within itself already?

Reply:
Parsing with a union ensures that if we get something that's _neither_ a string nor a number, the parse will fail instead of doing something we didn't want (like converting null to 0).